app/_hub/seifchen/kong-path-whitelist/index.md

```diff
@@ -0,0 +1,84 @@
+---
+name: Kong Path White List
+publisher: Seifchen
+
+categories:
+  - security
+
+type: plugin
+
+desc: Determine if the path is in the path whitelist, if not, return 403
+
+description: |
+  Add Kong Path White List to a Service or a Route or consumer with paths. The plugin will check the path is in the path whitelist, if not return 403.
+
+support_url: https://github.com/seifchen/kong-path-whitelist/issues
+
+source_url: https://github.com/seifchen/kong-path-whitelist
+
+license_type: Apache-2.0
+
+kong_version_compatibility:
+  community_edition:
+    compatible:
+      - 2.0.x
+      - 1.5.x
+      - 1.4.x
+      - 1.3.x
+      - 1.2.x
+    incompatible:
+      - 1.1.x
+      - 1.0.x
+      - 0.15.x
+      - 0.13.x
+      - 0.12.x
+      - 0.11.x
+      - 0.10.x
+      - 0.9.x
+      - 0.8.x
+      - 0.7.x
+      - 0.6.x
+      - 0.5.x
+      - 0.3.x
+      - 0.2.x
+
+params:
+  name: kong-path-whitelist
+  api_id: false
+  service_id: true
+  consumer_id: true
+  route_id: true
+  protocols: ["http", "https"]
+  dbless_compatible: yes
+
+  config:
+    - name: white_paths
+      required: yes
+      default: []
+      value_in_examples: ["/api/services", "/api/routes"]
+      description: The request path not match this path will forbidden with 403 code
+    - name: regex
+      required: true
+      default: true
+      value_in_examples:
+      description: boolean, if true will use ngx.re.match to match the request_path and white_paths,if false, Will strictly judge whether the two path are equal
```

Comment:
```suggestion
      description: A boolean value that specifies whether the plugin uses regex for path matching. If `true`, the plugin will use `ngx.re.match` to match the `request_path` and `white_paths` values. If `false`, it will strictly judge whether the two paths are equal.
```